Update example for disabling `buildAvailable`

Frank-Steiner opened the pull request:
The json code doesn't work (i.e. not disable the update checks) unless changing LabApp to ServerApp and removing the comma after the last "false"

<!--
Thanks for contributing to JupyterLab!
Please fill out the following items to submit a pull request.
See the contributing guidelines for more information:
https://github.com/jupyterlab/jupyterlab/blob/main/CONTRIBUTING.md
-->

## References
#18010 

## Code changes
The settings must be done for ServerApp, not LapApp, so the keyword has been changed. And the comma after the last "false" causes a syntax error, so that the jupyter_server_config.json file is ignored. So the comma was removed.

danyeaw reviewed (approved):
Thanks @Frank-Steiner!

krassowski commented:
I will rebase this on top of `main`.

krassowski committed:
Update directories.md

The json code doesn't work (i.e. not disable the update checks) unless changing LabApp to ServerApp

Co-authored-by: Frank-Steiner <fstboard-github@bio.ifi.lmu.de>
